react-web/src/components/atoms/typography/link/index.tsx

```diff
@@ -0,0 +1,50 @@
+import classNames from 'classnames'
+
+import { FontSize } from 'components/enums/font-size'
+
+import { ITypographyDefaultProps } from '..'
+import styles from './styles.module.scss'
+
+export enum TypographyTextDecoration {
+  underline = 'underline',
+  none = 'none',
+}
+
+export interface ILinkProps extends ITypographyDefaultProps {
+  /**
+   * Color for the link text
+   */
+  color?: string
+  /**
+   * Boolean for adding a text-decoration to the link text
+   */
+  textDecoration?: TypographyTextDecoration
+  /**
+   * The font size for the link text
+   */
+  fontSize?: FontSize
+}
+
+const Link = ({
+  text = 'Link',
```

Comment:
It would be better to leave the default value blank, just like you did in the label component.